models/campaign.tsx

```diff
@@ -0,0 +1,49 @@
+import { CampaignStatus } from 'constants/Donation'
+import ChevronRight from '@mui/icons-material/ChevronRight'
+import {
+  GridActionsCellItem,
+  GridColDef,
+  GridRowParams,
+} from '@mui/x-data-grid'
+
+// GridActionsCellItem seems to be badly defined, I can't set the
+// properties described here: https://mui.com/x/react-data-grid/column-definition/#special-properties
+// So I've created this monstrosity. Forgive me.
+const UntypesafeBox = (props: any) => {
+  return <GridActionsCellItem {...props} />
+}
```

Comment:
rip i also tried but didn't work. their examples on their docs also just used it like this